bank-link-sdk/src/main/java/com/aerosync/bank_link_sdk/StreamEvent.kt

```diff
@@ -1,6 +1,25 @@
 package com.aerosync.bank_link_sdk
 
-class StreamEvent(
-    val type: String,
-    val payload: Any
-) {}
\ No newline at end of file
+enum class WidgetEventType(val event: String) {
+    WIDGET_PAGE_SUCCESS("pageSuccess"),
+    WIDGET_PAGE_LOADED("widgetPageLoaded"),
+    WIDGET_CLOSE("widgetClose"),
+    WIDGET_ERROR("widgetError");
+    companion object {
+        fun fromEvent(event: String): WidgetEventType? {
+            return values().find { it.event == event }
+        }
+    }
+}
+
+data class PayloadSuccessType(
+    val user_id: String,
+    val user_password: String,
+    val ClientName: String,
```

Comment:
mixing casings :/

Reply:
@mark-aeropay : We have fixed the casing for the 2.0 version-  AeroPass flow.